hypothesis-python/src/hypothesis/extra/codemods.py

```diff
@@ -103,7 +103,7 @@ class HypothesisFixComplexMinMagnitude(VisitorBasedCodemodCommand):
     METADATA_DEPENDENCIES = (cst.metadata.QualifiedNameProvider,)
 
     @m.call_if_inside(
-        m.Call(metadata=match_qualname("hypothesis.strategies.complex_numbers"))
+        m.Call(metadata=match_qualname("hypothesis.strategies.complex_numbers"))  # type: ignore
```

Comment:
This is a mypy bug, confusing `m.Call` with `builtins.Call` (which does't accept a `metadata=` kwarg).

```
hypothesis/extra/codemods.py:106: error: Unexpected keyword argument "metadata" for "Call"
...python3.8/site-packages/mypy/typeshed/stdlib/builtins.pyi:95: note: "Call" defined here
```